po/tr.po

```diff
@@ -1005,13 +1005,12 @@ msgid "No item selected!"
 msgstr "Hiçbiri seçilmedi!"
 
 #: src/itemlistformaction.cpp:107
-#, fuzzy
 msgid "Do you really want to delete all articles (y:Yes n:No)? "
-msgstr "`%s' üzerine yazmak istediğinize emin misiniz (e:Evet h:Hayır)? "
+msgstr "Tüm yazıları silmek istediğinize emin misiniz (e:Evet h:Hayır)? "
 
 #: src/itemlistformaction.cpp:220 src/itemviewformaction.cpp:412
 msgid "Toggling read flag for article..."
-msgstr "Bayrak okuma açılıyor/kapatılıyor..."
+msgstr "Yazı için okundu bayrağı koyuluyor/kaldırılıyor..."
```

Comment:
Our of curiosity: why is there a slash? Gettext allows us to perform a few tricks with contexts and plurals, and we can just rephrase the message if the current one is very hard to translate unambiguously.

Reply:
There is no need for rephrasing. It's just that there is no single phrase that translates "toggle" in Turkish, so we generally use "on/off" in translation (as it can be seen in other parts of the messages). However, in this context it does not make sense to use on/off, since flags are generally put/pulled, so it is translated in such sense here.

Reply:
Got it, thanks! :+1: